dashboard/db/schema.rb

```diff
@@ -168,6 +168,18 @@
     t.string "callout_text"
   end
 
+  create_table "cap_user_events", charset: "utf8mb3", collation: "utf8mb3_unicode_ci", force: :cascade do |t|
+    t.string "name", limit: 64, null: false
+    t.datetime "occurred_at", null: false
```

Comment:
Now that I see the schema as a whole, is the `occurred_at` property redundant when we have the `timestamps` and the `created_at` property already?

Reply:
It definitely is redundant. I will update this.